src/Maths/math.vector.ts

```diff
@@ -3125,7 +3125,7 @@ export class Quaternion {
     /**
      * Sets the given vector3 "result" with the Euler angles translated from the current quaternion
      * @param result defines the vector which will be filled with the Euler angles
-     * @param order is a reserved parameter and is ignore for now
+     * @param order is a reserved parameter and is ignored for now

```

Comment:
@deltakosh Just noticed this function doesn't even accept `order` at all. Can I just drop this param in the JSDoc then? ;)

Reply:
Yes! thanks for that

Reply:
Done
3fbcbc4b5a00ffe3c3054d07fabe891397adc5db